doc/source/dev/contributor/adding_new/new_stats_distribution.rst.inc

```diff
@@ -0,0 +1,93 @@
+Adding A New Statistics Distribution
+------------------------------------
+
+For hundreds of years statisticians, mathematicians and scientists have needed
+to understand, analyze and model data.
+This has led to a plethora of statisics distributions, many of which are
+related to others.
+Modeling of new types of data continues to give rise to new distributions,
+as does theoretical considerations being applied to new disciplines.
+SciPy models about a dozen discrete distributions
+:ref:`discrete-random-variables` and 100 continuous distributions
+:ref:`continuous-random-variables`.
+
+To add a new distribution, a good reference is needed.
+Scipy typically uses [JKB]_ as its gold standard, with WikipediaDistributions_
+articles often providing some extra details and/or graphical plots.
+
+How to create a new continuous distribution
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are a few steps to be done to add continuous distribution XXXXX to SciPy.
+(Adding a discrete distribution is similar).
+
+
+Before Implementation
+.....................
+
+1. See if it already been implemented (that saves a lot of effort!)
+
+  - It may have been implemented with a different name.
+  - It may have been implemented with a different parameterization (shape parameters).
+  - It may be a specialization of a more general family of distributions.
+
+It is very common for multiple disciplines to discover/rediscover a
+distribution (or a specialization or different parameterization).
+There are a few existing SciPy distributions which are specializations
+of other distributions.  E.g. The :class:`scipy.stats.arcsine` distribution
+is a specialization of the :class:`scipy.stats.beta` distribution.
+These duplications exist for (very!) historical and widespread usage reasons.
+At this time, adding new specializations/reparametrizations of existing
+distributions to SciPy is not supported,  mainly due to the increase in user
+confusion resulting from such additions.
+
+2. Create a SciPy Issue on github listing the distribution, references and
+   reasons for its inclusion.
+
+
+Implementation
+..............
+
+#. Find an already existing distribution similar to ``XXXXX``.
+   Use its code as a template for ``XXXXX``.
+#. Read the docstring for class ``rv_continuous`` in
+   ``scipy/stats/_distn_infrastructure.py``.
+#. Write the new code for class ``xxxx_gen`` and insert it into
+   ``scipy/stats/_continuous_distns.py``,  which is in (mostly) alphabetical
+   order by distribution name.
+#. Does the distribution have infinite support? If not, left and/or right
+   endpoints ``a``, ``b`` need to be specified in the call to ``xxxx_gen()``.
+#. If the support depends upon the shape parameters,
+   ``xxxx_gen._get_support()`` needs to be implemented.
+#. The default ``_argcheck()`` implementation checks that the shape parameters
+   are positive.  Create a more appropriate implementation.
+#. If ``xxxx_gen.ppf()`` is expensive to compute relative to
```

Comment:
This comment might confuse a new developer, maybe add it to a section like "Additional points to keep in mind" / "Advanced topcis"? (I had to check the documentation https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.rv_continuous.html as I was not aware of this feature)

Reply:
It is probably good as is I think - it may require a bit of reading up, but that's not a bad thing. If it's separated out in an "advanced topics" list, then it sounds like things to ignore or deal with after merging (which usually turns into never).